zeit/next-site
# fix: the showcase section on homepage for iPhone 5/SE

## PR

### DESCRIPTION
The homepage of website is broken on iPhone 5/SE(320px), basically on all sizes below 330px, because the images in **Who’s Using Next.js** section are given minimum width of 330px due to which they are moving out of the normal display on sizes below 330px.


### SCREENSHOT(S) _iPhone(5/SE)_
#### Fix(Expected)
<img width="400" alt="expected" src="https://user-images.githubusercontent.com/19193724/82127178-4206fc80-97cf-11ea-99fa-40cbc039cf33.png">


#### Bug(Actual)
<img width="400" alt="actual" src="https://user-images.githubusercontent.com/19193724/82127184-4af7ce00-97cf-11ea-8d53-75162b8aecb6.png">


### HOW TO TEST

- Go to home page(http://localhost:3000)
- Scroll down to the Who’s Using Next.js section
- The image should not move out of the screen horizontally

Fixes #696
